pages/history.tsx

```diff
@@ -1,5 +1,135 @@
+          History
+        </Typography>
+        {!isMobileView && (
+          <Grid2 ml="auto" mr={6}>
+            <Button variant="outlined" sx={{ width: '100%' }}>
+              Export To Excel
+            </Button>
+          </Grid2>
+        )}
+      </Grid2>
+
+      {/* Filter views -- search, category dropdown, date range, internal only checkbox */}
+      <Grid2
+        container
+        xs={12}
+        direction={isMobileView ? 'column-reverse' : 'row'}
+        gap={isMobileView ? 2 : 4}
+        sx={{ px: 2, alignItems: 'center' }}
+      >
+        <Grid2 xs={12} md={2}>
+          <SearchField />
+        </Grid2>
+        <Grid2 xs={12} md={2}>
+          <Autocomplete
+            value={
+              categories.filter(
+                (category) =>
+                  category.name === (router.query.category as string)
+              )[0]
+            }
+            options={categories}
+            renderInput={(params) => <TextField {...params} label="Category" />}
+            isOptionEqualToValue={(option, value) => option._id === value._id}
+            getOptionLabel={(category) => category.name}
+            onChange={(_e, category) => {
+              updateQuery(router, 'category', category?.name)
+            }}
+          />
+        </Grid2>
+        <Grid2 xs={12} md={4} direction="row">
+          <Box
+            sx={{ display: 'flex', flexDirection: 'row', alignItems: 'center' }}
+          >
+            <DatePicker
+              label="Start Date"
+              renderInput={(params) => <TextField {...params} fullWidth />}
+              onChange={(date) => {
+                const startDate = new Date(date as string)
+                updateQuery(
+                  router,
+                  'startDate',
+                  startDate.toISOString().split('T')[0]
+                )
+              }}
+              value={router.query.startDate || null}
+            />
+            <Box ml={1} mr={1}>
+              <HorizontalRuleIcon sx={{ color: theme.palette.grey['500'] }} />
+            </Box>
+            <DatePicker
+              label="End Date"
+              renderInput={(params) => <TextField {...params} fullWidth />}
+              onChange={(_e, date) => {
+                const endDate = new Date(_e as string)
+                updateQuery(
+                  router,
+                  'endDate',
+                  endDate.toISOString().split('T')[0]
+                )
+              }}
+              value={router.query.endDate || null}
+            />
+          </Box>
+        </Grid2>
+
+        <Grid2 container md={2} direction="row" sx={{ alignItems: 'center' }}>
+          <Checkbox
+            onChange={(_e, checked) => {
+              if (checked) updateQuery(router, 'internal', 'true')
+              else updateQuery(router, 'internal', undefined)
+            }}
+            value={router.query.internal}
+          />
+          <Typography>Internal only</Typography>
```

Comment:
perhaps on mobile make the categories autocomplete half width and then the internal only on the same line as that.

Reply:
updated